packages/core/js-sdk/src/admin/inventory-item.ts

```diff
@@ -0,0 +1,140 @@
+import {
+  DeleteResponse,
+  FindParams,
+  HttpTypes,
+  SelectParams,
+} from "@medusajs/types"
+import { Client } from "../client"
+import { ClientHeaders } from "../types"
+
+export class InventoryItem {
+  private client: Client
+  constructor(client: Client) {
+    this.client = client
+  }
+
+  async create(
+    body: HttpTypes.AdminCreateInventoryItem,
+    query?: HttpTypes.SelectParams,
+    headers?: ClientHeaders
+  ) {
+    return await this.client.fetch<HttpTypes.AdminInventoryItemResponse>(
+      `/admin/inventory-items`,
+      {
+        method: "POST",
+        headers,
+        body,
+        query,
+      }
+    )
+  }
+
+  async update(
+    id: string,
+    body: HttpTypes.AdminUpdateInventoryItem,
+    query?: SelectParams,
+    headers?: ClientHeaders
+  ) {
+    return await this.client.fetch<HttpTypes.AdminInventoryItemResponse>(
+      `/admin/inventory-items/${id}`,
+      {
+        method: "POST",
+        headers,
+        body,
+        query,
+      }
+    )
+  }
+
+  async list(
+    query?: FindParams & HttpTypes.AdminInventoryItemFilters,
```

Comment:
question: Doesn't the type already extend the `FindParams`?
```suggestion
    query?: FindParams & HttpTypes.AdminInventoryItemFilters,
```

Reply:
good catch, I will fix it